_posts/2019-03-06-cockpit-189.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 189
+author: Gundersanne
+date: 2019-3-06
```

Comment:
Not sure if it matters somewhere, but let's please make this "03" for ISO format.